contracts/Tokens/VAI/VAI.sol

```diff
@@ -15,7 +15,7 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-pragma solidity ^0.5.16;
+pragma solidity 0.8.13;
```

Comment:
This contract is not upgradable. I wouldn't do this change in the source code.

Reply:
Sorry, my bad. I will undo the changes.